packages/fhir-router/src/repo.ts

```diff
@@ -305,15 +310,18 @@ export class MemoryRepository extends BaseRepository implements FhirRepository {
     return {
       resourceType: 'Bundle',
       type: 'history',
-      entry: ((this.#history?.[resourceType]?.[id] ?? []) as T[])
+      entry: ((this.history.get(resourceType)?.get(id) || []) as T[])
```

Comment:
Just curious, was there a reason why there was a change from using `??` to `||`. It seems like it shouldn't affect the behavior and should be fine as is.

Reply:
Good question.  `??` is technically more correct, but it's not natively supported by our target JavaScript version, so it gets transpiled to more complex code.

See: https://www.typescriptlang.org/play?ts=5.0.4#code/GYVwdgxgLglg9mABAEziARgGwKYEUTYDOsCAsgIYBOA1gBQxgAOIUAXIsZQwOaIA+icMmzAG2ZAEpEAbwBQiRBASE4OAHSY43ekxaIA-PsQByYcHIhMUYxIDcsgL6zZoSCSSoMOAAoxG2HWY2DiguMF4BIRExSRl5RWVVbA0tQL0+AVMRCysbewcgA

![image](https://user-images.githubusercontent.com/749094/232096242-a61adc67-9bbe-4811-8f08-b189898542d6.png)

Input:

```ts
function doubleQuestionMark(input: string | undefined) {
  console.log(input ?? 'default');
}

function doublePipe(input: string | undefined) {
  console.log(input || 'default');
}
```

Output:

```js
"use strict";
function doubleQuestionMark(input) {
    console.log(input !== null && input !== void 0 ? input : 'default');
}
function doublePipe(input) {
    console.log(input || 'default');
}
```

In some cases, like this sample code when `input` is a `string`, then the behavior is subtly different when `input` is empty string `''`.

In the PR above, the value is always an `Object`, so `||` and `??` have identical behavior.

By using `||` our bundle size is slightly smaller and the runtime is slightly faster.  This is probably a silly micro optimization.